spec/ScorecardSpec.js

```diff
@@ -0,0 +1,78 @@
+describe("Scorecard", function () {
+  var scorecard
+  var f1
+  var f2
+  var f3
+  beforeEach(function () {
+    scorecard = new Scorecard
+    f1 = new Frame
+    f2 = new Frame
+    f3 = new Frame
+    spyOn(f1, 'setNumber')
+    spyOn(f2, 'setNumber')
+    spyOn(f3, 'setNumber')
+    spyOn(f1, 'score')
+    spyOn(f2, 'score')
+    spyOn(f3, 'score')
+  })
```

Comment:
Perfect unit testing with spies! Definitely something that I need to work on :')